addons/common/functions/fnc_displayTextStructured.sqf

```diff
@@ -1,11 +1,12 @@
 /*
- * Author: commy2, Glowbal
+ * Author: commy2, Glowbal, GitHawk
  * Display a structured text.
  *
  * Arguments:
  * 0: Text <ANY>
  * 1: Size of the textbox (default: 1.5) <NUMBER>
  * 2: Target Unit. Will only display if target is the player controlled object (default: ACE_player) <OBJECT>
+ * 3: Custom Width <NUMBER><OPTIONAL>
```

Comment:
I guess it doesn't hurt but it will make it inconsistent with other hints. @commy2 ?